ansible/ansible-playbook.yml

```diff
@@ -0,0 +1,110 @@
+- name: Update web servers
+  hosts: managed-nodes
+  remote_user: ec2-user
+  vars:
+    host: "{{ inventory_hostname }}"
+  vars_files:
+    - .ansible-secrets.yml
+
+  tasks:
+    - name: Install Git
+      yum:
+        name: "git"
+        state: "present"
+        update_cache: yes
+      become: yes # Installing git must be run as the root user
+
+    - name: Install Python
+      yum:
+        name: "python3.11"
+        state: "present"
+        update_cache: yes
+      become: yes # Installing python must be run as the root user
+
+    - name: Install Node
+      yum:
+        name: "nodejs"
+        state: "present"
+        update_cache: yes
+      become: yes # Installing node must be run as the root user
+
+    - name: Install Poetry
+      shell:
+        cmd: "curl -sSL https://install.python-poetry.org | python3.11 -"
+        creates: "/home/ec2-user/.local/bin/poetry"
+
+    - name: Create folder for Playlist Manager
+      file:
+        path: /opt/playlist-manager
+        state: directory
+        owner: "ec2-user"
+      become: yes
+
+    - name: Clone Playlist Manager
+      git:
+        repo: "https://github.com/CalPinSW/playlist-manager.git"
+        version: "add-ansible-configuration"
+        dest: "/opt/playlist-manager"
+        force: true
+
+    - name: Install Project Backend Dependencies
+      shell:
+        chdir: "/opt/playlist-manager/backend"
+        cmd: "poetry env use python3.11; poetry install"
+
+    - name: Install Project Frontend Dependencies
+      shell:
+        chdir: "/opt/playlist-manager/frontend"
+        cmd: "npm install"
+
+    - name: Fetch backend environment template
```

Comment:
You're the second person recently to use this pattern of pulling the template before populating it which I'd never seen before, but I really like! 😄